cmd/kubelet/app/server.go

```diff
@@ -902,6 +902,9 @@ func parseResourceList(m utilconfig.ConfigurationMap) (api.ResourceList, error)
 			if err != nil {
 				return nil, err
 			}
+			if q.Amount.Sign() == -1 {
+				return nil, fmt.Errorf("resource for %q has negative quantity: %v", k, v)
```

Comment:
Maybe rephrase to better indicate it's an error (resource must be greater than or equal to zero, resource cannot be negative, something like that)?